app/controllers/v1/people_controller.rb

```diff
@@ -1,7 +1,7 @@
 class V1::PeopleController < V1::BaseController
 
   def create
-    @person = PersonWithRemoteFields.find_or_build(person_params)
+    @person = PersonConstructor.new(person_params).build
```

Comment:
I think `PersonConstructor.build(person_params)` looks fine. It makes implementation harder, though, because then you can't use instance variables. It would mean the whole class needs to be rewritten...